pkg/autoupdate/tuf/library_lookup.go

```diff
@@ -0,0 +1,75 @@
+package tuf
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+
+	"github.com/kolide/launcher/pkg/autoupdate"
+)
+
+// CheckOutLatest returns the path to the latest downloaded executable for our binary, as well
+// as its version.
+func CheckOutLatest(binary autoupdatableBinary, rootDirectory string, updateDirectory string, channel string) (string, string, error) {
```

Comment:
I always find >2 return values confusing, consider naming the return values
```suggestion
func CheckOutLatest(binary autoupdatableBinary, rootDirectory string, updateDirectory string, channel string) (path string, version string, err error) {
```

Reply:
I agree that more than 2 return values is confusing, but I think naming them is an antipattern. (Though there's disagreement in go)

It looks pretty, but it creates them as implicit vars, which always feels like it breaks scoping to me